epub33/core/index.html

```diff
@@ -4947,22 +5141,28 @@ <h5>Other <code>nav</code> Elements</h5>
 								<code>nav</code> elements: they MAY represent navigational semantics for any information
 							domain, and they MAY contain link targets with homogeneous or heterogeneous semantics.</p>
 
-						<aside class="example">
-							<p>The following example shows a custom list of tables navigation element.</p>
-
-							<pre>&lt;nav aria-labelledby="lot"&gt;
-    &lt;h2 id="lot"&gt;List of tables&lt;/h2&gt;
-    &lt;ol&gt;
-        &lt;li&gt;&lt;span&gt;Tables in Chapter 1&lt;/span&gt;
-            &lt;ol&gt;
-                &lt;li&gt;&lt;a href="chap1.xhtml#table-1.1"&gt;Table 1.1&lt;/a&gt;
-                &lt;/li&gt;
-                &lt;li&gt;&lt;a href="chap1.xhtml#table-1.2"&gt;Table 1.2&lt;/a&gt;&lt;/li&gt;
-            &lt;/ol&gt;
-        &lt;/li&gt;
-    	…
-    &lt;/ol&gt;
-&lt;/nav&gt;</pre>
+						<aside class="example" title="A list of tables navigation element">
```

Comment:
```suggestion
						<aside class="example" title="A list of navigation elements for tables">
```

Reply:
The LOT semantic is "list of tables". I could maybe single quote this, or capitalize, to make it clearer. I find the rewrite less clear as there's only one element.

Reply:
Hm, where or where did the epub:type go in this example? I assumed it was there.

We used to have epub:type="lot" on the nav element up until 3.1 when it got replaced by `aria-labelledby`, but that makes it violate the recommendation that other nav types have an `epub:type` attribute. I'm going to restore.

Reply:
Ah! Now I understand, but it was really not clear (to me). I think putting the "list of tables" in a quote might help a lot. (I would probably use a double quote and not a single quote, but that is a tiny detail)

Reply:
> I would probably use a double quote and not a single quote

Ya, it was double quoted in the attribute, but I guess I could have use the quot character. What I did, instead, was tweak the title to be "Adding a custom navigation element" and then described that it's a list of tables so I could quote it properly.

Reply:
You are a man of compromise :)

I am fine to merge.